tsconfig.json

```diff
@@ -1,11 +1,19 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "baseUrl": ".",
+    "declaration": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "skipLibCheck": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
Will you fix the "any" things as part of the POC?

Reply:
I will refactor lots of stuff as soon as @ndelangen and whoever wants to join the discussion are happy with the stuff happening here 🙂

Reply:
In some cases you can use type `unknown` it is like `any` but type-safe.
[new `unknown` type in TS 3+](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type)

Reply:
Yes, it is better not to use `any`.
And for `unkown is not defined` - maybe you have an old version of TS installed localy? Or made a typo and defined it as a variable and not a type?

Reply:
`noImplicitAny` is true again